package.json

```diff
@@ -10,13 +10,20 @@
     "url": "https://github.com/react-native-community/react-native-share/issues"
   },
   "main": "index.js",
+	"types": "index.d.ts",
   "files": [
     "android",
     "components",
     "ios",
     "windows",
     "RNShare.podspec"
   ],
+	"peerDependencies": {
+		"@types/react": "*",
+		"@types/react-native": "*",
+		"react": "*",
+		"react-native": "*"
+	},
```

Comment:
Why do we need these declarations? This would work without it?

Reply:
I copied and pasted them from the commit in another React native project that added TS types. Now removed.